### ISSUE TITLE
Comments get lost when generating annotations

### ISSUE THREAD

**Issue by cmer** [ctran/annotate_models#951]
When there are already comments above a class definition, AnnotateModels deletes them when regenerating annotations.

Here's a short video showing the issue: https://cln.sh/Iw7NsPk7yEfmkyPWtIim

## Version

- annotate version: 3.2
- rails version: 5.2.6.2
- ruby version: 2.7.5

**Comment by joevandyk** [comment]
running into this as well

**Comment by alexeyr-ci1** [comment]
It works to add a blank line above the comment before annotating, but you need to find them first (I spotted it accidentally when reviewing a PR).

**Comment by cmer** [comment]
Yes but obviously nobody remembers to do that on a large team :)

**Comment by alexeyr-ci1** [comment]
Sorry, it's actually worse than I thought. It looked like it worked only because the comment wasn't getting regenerated. If I use `--force` (or do a schema change which forces a regeneration), then the first time eats the empty line and leaves the comment intact (so really hard to note in code review), and the next time eats the comment 🤣.

**Comment by filipesperandio** [comment]
Is there a workaround for this?

**Comment by drwl** [comment]
Hey folks, I believe I was able to solve this bug in my gem https://github.com/drwl/annotaterb/pull/72

**Comment by ghost** [comment]
I wasn't aware of the fork. I'll definitely check it out. Thanks!

**Comment by fiedl** [comment]
> It works to add a blank line above the comment before annotating

For me, this works only the first time. When I run `annotate` the second time, the comments are being removed.

How do I need to wrap or format the existing class documentation to protect it?

**Comment by dben89x** [comment]
Yeah this is absolutely horrible. I lost so many comments that were not part of a commit. Why in the world would this not just prepend to a file?
